src/Files.App/Actions/Show/ToggleDotFilesSettingAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal sealed class ToggleDotFilesSettingAction : ObservableObject, IToggleAction
+	{
+		private readonly IFoldersSettingsService settings;
```

Comment:
It'd be better to be like this: 
```suggestion
		private readonly IFoldersSettingsService FoldersSettingsService;
```